sympy/printing/pycode.py

```diff
@@ -40,7 +40,7 @@
     'floor': 'floor',
     'gamma': 'gamma',
     'hypot': 'hypot',
-    'loggamma': 'lgamma',
+    'loggamma': 'loggamma',
```

Comment:
This is used for the math module, so should not be changed here.

Reply:
ok ok